### DISCUSSION TITLE
chat_join_request_handler passes a truncated invite_link

### DISCUSSION THREAD

**Discussion by fttx** [eternnoir/pyTelegramBotAPI#1859] [Q&A]
1. What version of pyTelegramBotAPI are you using? **4.8.0**

2. What OS are you using? **macOS**

3. What version of python are you using? **3.10.8**

The following code

```python
@bot.chat_join_request_handler()
def chatJoin(info):
    print("chatJoin " + info.invite_link.invite_link)
```

produces this result:

![image](https://user-images.githubusercontent.com/5663423/210165370-24ee0876-3511-4b2e-9d8f-a54866d62325.png)

Basically the link is truncated with ellipsis. Is there a way to obtain the full URL?

**Comment by coder2020official** [comment]
This is behaviour of telegram:


invite_link | String | The invite link. If the link was created by another chat administrator, then the second part of the link will be replaced with “…”.
-- | -- | --

**Comment by fttx** [comment]
Thank you for the fast reply! Do you think it is safe to use the first part as a unique identifier to find the full link in a local database?

**Comment by Badiboy** [comment]
Nobody knows except Telegram developers.

**Comment by coder2020official** [comment]
I think you can use it, but you'll need to store creator of the link to be more reliable.  Not sure though.

**Comment by coder2020official** [comment]
But it won't be 100% safe.
